lib/publiccloud/provider.pm

```diff
@@ -556,14 +556,14 @@ Wrapper around C<<$self->vault_api()>> to get retry capability.
 sub vault_api {
     my ($self, $path, %args) = @_;
     my $ret;
-    my $tries = 3;
+    my $tries = 10;
```

Comment:
```suggestion
    my $tries = get_var(PUBLICCLOUD_VAULT_TRIES, 3);
```

Reply:
yet another 2 new variables? why not deciding in default values for all jobs? These type of failures might happen for any provider, any test... we would need to add the same variable/value to all jobs anyway. What is the difference?

Reply:
I agree with Jose.